include/roboteam_utils/Print.h

```diff
@@ -0,0 +1,48 @@
+#ifndef RTT_ROBOTEAM_AI_SRC_UTILITIES_PRINT_H_
+#define RTT_ROBOTEAM_AI_SRC_UTILITIES_PRINT_H_
+
+#include <iostream>
+#include <iomanip>
+
+class Printer {
+ public:
+  static void print(const std::string& color, const std::string& type, const std::string& file, const std::string& func, int line, const std::string& txt) {
+      auto fileTxt =  file + ":" + std::to_string(line);
```

Comment:
why not just feed this to the stream instead of converting it to an std::string first

Reply:
```suggestion
```